x-pack/plugins/triggers_actions_ui/public/application/sections/rules_list/components/rule_snooze/recurrence_scheduler/recurrence_scheduler.scss

```diff
@@ -0,0 +1,8 @@
+.RecurrenceScheduler {
```

Comment:
I'm not able to do `.ramRecurrenceScheduler__labelWrapper | .ramRecurrenceScheduler__fieldWrapper` because I can't apply custom class names to internal `EuiFormRow` components. This CSS is for overriding those internal styles.